src/CSS/Create.scss

```diff
@@ -34,7 +43,9 @@ $color : #d04c55;
 .portfolio-experiment {
   background: #efdccc;
   border-radius: 50%;
+  border: 2px solid #d04c55;
   box-shadow: 0 0 .5rem 0 gray;
+  margin-top: 100%;
```

Comment:
don't use % for margins: if your container size increases (e.g. you just add an element in the div), the design will break. prefer rem:

    margin-top: 1rem;

if you need more than 4rem, that's a layout problem: use flexbox/grid ;)